greymatter/variables.tf

```diff
@@ -30,14 +12,10 @@ variable "key_pair_name" {
   default = "enter-ecs"
```

Comment:
This probably isn't a great default...it just happens to be the key pair name I have, I think it slipped through one of my PRs.